docs/content/providers/kubernetes-crd.md

```diff
@@ -177,26 +177,32 @@ _Optional,Default: empty (process all resources)_
 
 ```toml tab="File (TOML)"
 [providers.kubernetesCRD]
-  labelselector = "A and not B"
+  labelselector = "foo"
```

Comment:
I don't think this is a valid labelselector, and is not indicative of what would be used.

I would recommend:

```
labelSelector = "app=traefik
```

As this is what is used in our example code below

Reply:
Hey @dtomcej 
While i agree that is not indicative, it is a valid labelSelector which only do label key matching.
Suggestion applied !

Reply:
I didn't know you could do bare label key matching! TIL!